packages/libs/eda/src/index.tsx

```diff
@@ -68,6 +68,9 @@ const singleAppMode = process.env.REACT_APP_SINGLE_APP_MODE;
 const showUnreleasedData =
   process.env.REACT_APP_SHOW_UNRELEASED_DATA === 'true';
 
+const enablePublicUserDatasets =
+  process.env.COMMUNITY_DATASETS_ENABLED === 'true';
```

Comment:
Developers will need 
```
COMMUNITY_DATASETS_ENABLED=true
```
in their `packages/libs/eda/.env.local` to pass the flag through to the map app. Not sure what it does there, however...